.changeset/itchy-games-pretend.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/design-system-components": patch
```

Comment:
Question: should there be two change sets?

Reply:
Not sure of the etiquette here – I added [one for the tokens](https://github.com/hashicorp/design-system/pull/740/files#diff-e2e962f0d1fde94e1cd08f5c87a7229e92dab2895b894ca96fd3c6729014c602) and [one for the component](https://github.com/hashicorp/design-system/pull/740/files#diff-1a80ffc289fe128548c9d95fc57c51422a9dc2038dc4421d7ea231030ec91d04) because I wanted distinct (and hopefully more relevant) messages, but happy to merge them if that makes more sense.

Reply:
I am not entirely sure either. @Dhaulagiri can you clarify?

Reply:
I think the way @alex-ju has it here makes sense. In practice I think either approach works more or less the same, this just gives a little more control over the changelog message like Alex mentioned.

Reply:
Thank you both for answering my question! 👍